mod/groups/views/default/resources/elements/groups/alpha.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
I think we should reserve resource views to mostly complete pages. Why not just "groups/elements/alpha" etc.?